src/config_file.c

```diff
@@ -0,0 +1,239 @@
+/*
+ * Copyright (C) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ */
+#include <stdio.h>
+#include <unistd.h>
+#include <stdlib.h>
+#include <errno.h>
+#include <string.h>
+
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+
+#include "config_file.h"
+#include "lib/sedcli_log.h"
+
+#define SEDCLI_CONF_DELIM "="
+
+#define ARRAY_SIZE(x) (sizeof(x) / sizeof(x[0]))
+
+enum {
+	KMIP_IP = 0,
+	KMIP_PORT,
+	CLIENT_CERT,
+	CLIENT_KEY,
+	CA_CERT,
+	UNDEFINED
```

Comment:
If there are any plans of extending this enum in future, 
I would suggest introducing a ',' (comma) at the end of the list.

Reply:
Currently no plans to extend this enumeration